pypicloud/access/__init__.py

```diff
@@ -22,6 +23,8 @@ def includeme(config):
         dotted_name = RemoteAccessBackend
     elif dotted_name == 'sql':
         dotted_name = SQLAccessBackend
+    elif dotted_name == 'ldap' and LDAP_ENABLED:
+        dotted_name = LDAPAccessBackend
```

Comment:
If `ldap` isn't installed this will produce an unclear message (something about being unable to import the path). I recommend looking at how the [caching backend](https://github.com/mathcamp/pypicloud/blob/master/pypicloud/cache/__init__.py#L20) sets itself up (since it isn't guaranteed to have flywheel installed at runtime). It does a lazy import and [prints out a clear error message](https://github.com/mathcamp/pypicloud/blob/c289855f4251a816e94aacc7c52f07e82ba4aceb/pypicloud/cache/dynamo.py#L20) if it fails.